src/styles/variables.scss

```
@@ -0,0 +1,6 @@
+$blue: #00acdc;
```

It is better to use more universal naming for variables.

Example:

```suggestion
$main-color: #00acdc;
```
or
```suggestion
$prime-color: #00acdc;
```